src/Parser/Rogue/Outlaw/Modules/RogueCore/Energy.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import Wrapper from 'common/Wrapper';
+import resourceSuggest from 'Parser/Core/Modules/ResourceTracker/ResourceSuggest';
+
+import EnergyTracker from '../../../Common/Resources/EnergyTracker';
+
+class Energy extends Analyzer {
```

Comment:
As far as I can tell, this class only gives a suggestion involving CombatPotency. As such, it should probably be called CombatPotency instead of 'Energy'

Reply:
In other rogue specs there is one file that gives all energy suggestions... but outlaw only has one extra source!